content/1.7/back-office/dashboard/dashboard.md

```diff
@@ -0,0 +1,179 @@
+    - **Link to your Google Analytics account** - when pressed, redirects to the Module manager page.
+    - **Visits** - when pressed, redirects to the Stats page.
+    - **Unique visitors** - when pressed, redirects to the Stats page.
+    - **Traffic Sources** - below shows the sources by which the shop is reached, a circle graph below visualizes the statistics.
+
+## Lower central column 
+
+### Forecast
+
+Displays the year that the forecast is showing.
+
+#### Call to action
+
+Firstly 2 buttons are displayed:
+     
+ - **Arrow to the left** - when pressed, displays the forecast one year back.
+ - **Arrow to the right** - when pressed, displays the forecast one year up front.
+
+On the right of the block there are 2 buttons: 
+
+ - **Configure** - when pressed, the forecast graph turns to exact numbers of: year, traffic, conversion rate, average cart value, sales. Has all the months of the year listed, every month has input field for traffic, conversion rate (measured in percentages), average cart value (measured in default shop currency. And a "Save" button, which when pressed, reverts to the previous dashboard.
+ - **Refresh** - refreshes the stats of the Forecast.
+
+Below there are 4 different colors metric buttons that can be pressed to display the wanted graph accordingly below:
+
+ - **Traffic**
+ - **Convesrion**
+ - **Average Cart Value**
+ - **Sales**
+
+Below the graph shows graph of data by month and numbers.
+
+## Central column below 
+
+### Products and Sales
+
+#### Call to action
+
+ - **Configure** - when pressed the tab changes to a configuration where it's possible to edit the view of the products and sales, 4 fields are displayed:
+   - **Number of "Recent Orders" to display** - drop down field with the options: 5, 10, 20, 50.
+   - **Number of "Best Sellers" to display** - drop down field with the options: 5, 10, 20, 50.
+   - **Number of "Most Viewed" to display** - drop down field with the options: 5, 10, 20, 50.
+   - **Number of "Top Searches" to display** - drop down field with the options: 5, 10, 20, 50.
+   - "Save", when pressed, reverts to the previous product and sales dashboard.
+ - **Refresh** - refreshes the stats of the Products and Sales.
+
+Below there are 4 buttons that can be chosen:
+
+ - **Recent orders** - when this option is selected, below the last 10 orders are displayed. Options displayed with this selection. Also every order has a search button at the right side of every field, when pressed redirects to the order.
+    - **Customer name** - when pressed on a customers name, will redirect to the page - Manage your Customers.
+    - **Products** 
+    - **Total Tax excl.**
+    - **Date**
+    - **Status**
+ - **Best sellers**
+ - **Most viewed** - when this option is selected, by default it will show a message: "You must enable the "Save global page views" option from the "Data mining for statistics" module in order to display the most viewed products, or use the Google Analytics module."
```

Comment:
```suggestion
 - **Most viewed** - when this tab is selected and if the "Save global page views" option from the "Data mining for statistics" module is disabled, the following message is displayed: "You must enable the "Save global page views" option from the "Data mining for statistics" module in order to display the most viewed products, or use the Google Analytics module."
```

Reply:
Could you specify the columns of this table (image, product, category, total sold, sales, net profit) ?